### DISCUSSION TITLE
Modus [Silo laden] - Keilsilo wird nicht vollständig entladen

### DISCUSSION THREAD

**Discussion by Langer1234561** [Courseplay/Courseplay_FS22#3220] [Q&A]
Hallo zusammen,

da ich mir nicht sicher bin, ob es ein Setup-Problem ist oder ein Bug wollte ich nicht direkt einen Issue erstellen.

Ich habe den Standard-Radlader (Claas Torion) mit der Standar-Silagegabel (Albutt) im Standard-Keilsilo im Einsatz um die Standard-BGA (500 MW) zu befüllen. Wir spielen auf der NF-Marsch im Multipayer.

Eingestellt ist der Radlader wie folgt:
![Farming Simulator 22 Screenshot 2024 04 30 - 02 27 17 83](https://github.com/Courseplay/Courseplay_FS22/assets/128261480/150a05c2-e9fd-4d1d-8ad7-ef88bb6ba4bb)

Grundsätzlich funktioniert das Beladen der BGA auch, jedoch wird im Silo nicht bis nach ganz hinten gefahren. Der Radlader lädt ab einer gewissen Tiefe im Silo keine Silage mehr auf und fährt eine Endlossschleife auf der gleichen Stelle: [Video](https://1drv.ms/v/s!AkjSUGkb2LQchck9vqN_JQCNKFrSog?e=GxQi7n)

Im Anschluss habe ich die Gabel manuell befüllt und CP mit gleichen Einstellungen erneut gestartet. Hier wurde das Silo dann an einer anderen Position angefahren und problemlos in die BGA entladen: [Video](https://1drv.ms/v/s!AkjSUGkb2LQchck-tV3yvMg71T-KLg?e=Hh6Ypm)

Nach einiger Zeit fährt der Radlader erneut nicht weit genug in das Silo hinein, und die Gabel wird nicht bzw. nicht vollständig beladen: [Video](https://1drv.ms/v/s!AkjSUGkb2LQchclAx-2FfLB9XhDlRg?e=PkwMKs). Dieser Prozess wiederholt sich nun endlos. 

Die Frage ist nun, warum CP nicht weit genug in das Silo fährt bzw. die Spur im Silo wechselt. Hatte schon eimal jemand dieses Problem? Habe ich in den Einstellungen etwas falsch eingestellt? Werden weitere Infos benötigt?

Danke schon einmal!

**Comment by Tensuko** [comment]
Der Grund für den nicht gemachten Spurwechsel ist, dass im vordersten Bereich noch Silage auf dem Boden liegt.
Das Ziel ist also vorne links. Wenn du das vom Boden weg nimmst, geht es.
Sieht mir nach nicht richtig eingestellter Höhe der Schaufel aus.

Bis ganz nach hinten an die Wand geht nicht, das haben wir versucht anzupassen, aber dann bleibt er an der Wand stecken.

Das schlechte Ladeverhalten liegt an Giants.... Die Schaufeln haben auch nur Trigger, die bei Kontakt mit einem Haufen im richtigen Winkel den Haufen in die Schaufel übertragen.... ist halt echt Murks...

**Reply by Langer1234561** [reply]
Danke für die schnelle Rückmeldung! Ich habe ewig herumprobiert und es ist mir etwas peinlich, dass ich das nicht selbst herausgefunden habe.
Die Schaufel muss ich auf diese Höhe einstellen, da der Radlader sonst an der blöden Silokante des Standard-Silos hängenbleibt. Ich werde aber ein Modsilo nutzen bei der nächsten Ernte.
